src/js/node/crypto.ts

```diff
@@ -24,7 +24,26 @@ const {
   randomInt: _randomInt,
   pbkdf2: pbkdf2_,
   pbkdf2Sync: pbkdf2Sync_,
-} = $zig("node_crypto_binding.zig", "createNodeCryptoBindingZig");
+  hkdfSync: _hkdfSync,
+} = $zig("node_crypto_binding.zig", "NodeCrypto.create");
+
+function hkdfSync(digest, ikm, salt, info, keylen) {
+  if (ikm instanceof KeyObject) {
+    ikm = ikm.export();
+  }
+
+  return _hkdfSync(digest, ikm, salt, info, keylen).buffer;
+}
+
+function hkdf(digest, ikm, salt, info, keylen, callback) {
+  if (callback != null) {
+    // Causes weird test failures
+    // process.nextTick(() => callback(null, _hkdfSync(digest, ikm, salt, info, keylen)));
```

Comment:
For the sync version, we probably do need use nextTick, but I would suggest doing the work first and then enqueueing that if we do want to keep it synchronous for now